typescript/starter/prisma/schema.prisma

```diff
@@ -0,0 +1,20 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
```

Comment:
How would you feel about this being sqlite instead of postgres? That would make it easier to get started.

Reply:
This might sabotage my other intention of using this as a starting point for a Prisma Pulse guide -- but willing to consider adding a configuration step (switch sqlite → postgres) if it's a big deal though. wdyt?

Reply:
or another option would be to have a `typescript/starter` example using sqlite, and a `typescript/starter-postgres` using postgres?

Reply:
nah that's a fair point. Let's stick with postgres then.

I would also recommend adding a `README.md` explaining how to get started with this